src/helpers/git_helpers/rebase_helpers.sh

```diff
@@ -30,3 +30,11 @@ function rebase_in_progress {
     echo false
   fi
 }
+
+
+# Determines what branch is being rebased
+#
+# Only call when rebase_in_progress is true
+function rebasing_branch_name {
+  sed 's/^refs\/heads\///' < .git/rebase-apply/head-name
```

Comment:
Also, would it make sense to do the check for `rebase_in_progress` here then, if it only makes sense to call when that's true?